### PR TITLE
feat: Chartbump/nvidia gpu operator 23.3.1

### PR DESCRIPTION
**Pull request by faiq**
**What problem does this PR solve?**:
Automated chartbump 

**Which issue(s) does this PR fix?**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [ ] If the PR adds a version bump, ensure there is no breaking change in Licensing model (or NA).
- [ ] If a chart is changed or app configuration is significantly changed, the chart version is correctly incremented (so that apps are not automatically upgraded from a previous version of DKP).

### CODE REVIEW

File: services/nvidia-gpu-operator/23.3.1/defaults/cm.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        version: 3.1.7-1-ubuntu20.04
+    dcgmExporter:
+        enabled: true
+        version: 3.1.7-3.1.4-ubuntu20.04
+    validator:
+        # we are using a newer version of the validator pod that
+        # removes the compat libs from the container
+        # this can cause a mismatch in cuda versions
+        # see: https://github.com/NVIDIA/gpu-operator/issues/389#issuecomment-1215781064
+        # for more details
```
Comment (faiq) [inline services/nvidia-gpu-operator/23.3.1/defaults/cm.yaml:38]:
```suggestion
```

**Comment by gracedo** [comment]
not sure why the bot keeps removing the label, lol, but since we need changes in kommander with every nvidia bump now, we can add it to this list to auto apply the `open-kommander-pr` label https://github.com/mesosphere/kommander-applications/blob/f8a072512d0c2575c047ed230ca62b815a0952fa/.github/services-pr-labeler.yaml#L1-L19

**Comment by faiq** [comment]
@gracedo i added that enhancement in the nvidia automation pr https://github.com/mesosphere/kommander-applications/pull/1234/commits/9b389ce57609a27b4e2c49ff503c33ee3cec2486

File: services/nvidia-gpu-operator/23.3.1/defaults/cm.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nvidia-gpu-operator-23.3.1-d2iq-defaults
+  namespace: ${releaseNamespace}
+data:
+  values.yaml: |
+    nfd:
+        enabled: false
```
Comment (gracedo) [inline services/nvidia-gpu-operator/23.3.1/defaults/cm.yaml:9]: hmm, it looks like automation is using 4 spaces indentation? we generally use 2 in our yaml files, wondering if we could use 2 here

**Pull request closed without merging**